tests/test_parsing/test_parse_sdf.py

```diff
@@ -2,7 +2,7 @@
 
 
 def test_recursive_sdf_parsing():
-    parsed_sdf = load_sdf("mud_test.sdf")
+    parsed_sdf = load_sdf("mud_test/model.sdf")
     world = parsed_sdf.aggregate_order[0]
     assert len(world.includes) == 5
     assert len(world.models) == 7
```

Comment:
Can you add some asserts checking the injected paths are correct?

Reply:
+1